doc/articles/faq.md

```diff
@@ -2,39 +2,39 @@
 uid: Uno.Development.FAQ
 ---
 
-# FAQ: General
+# Frequently Asked Questions
 
 ## About Uno Platform
 
-#### What is the Uno Platform?
+### What is the Uno Platform?
 
-Uno Platform lets you write an application once in XAML and/or C#, and deploy it to [any target platform](getting-started/requirements.md). Read more [here](what-is-uno.md).
+Uno Platform lets you write an application once with C# and XAML, and deploy it to [any target platform](getting-started/requirements.md). Read more [here](what-is-uno.md).
```

Comment:
Did the "and/or" intentionally existed to refer to C# markup?

Reply:
The and/or phrasing is intentional, let's keep it this way for now.

Reply:
@jeromelaban XAML and/or C# however gives "precedence" to XAML a bit, wouldn't C# and/or XAML be better?

Reply:
I don't really think it matters. And we came from XAML first so it is 'fair' XAML gets mentioned. That said, depending on the context/wording, I wonder if we should use C# Markup more (and not just C#) as it could get mistaken for C# for back end logic.